mindmeld/components/entity_resolver.py

```diff
@@ -55,7 +55,14 @@ class EntityResolver:
     """The prefix of the ES index."""
 
     def __init__(
-        self, app_path, resource_loader, entity_type, es_host=None, es_client=None
+        self,
+        app_path,
+        resource_loader,
+        entity_type,
+        es_host=None,
+        es_username=None,
+        es_password=None,
+        es_client=None,
```

Comment:
just wonder is there any reason that we have `es_client` in constructor of Entity Resolver but not Question Answerer? Does the passed-in `es_client` take precedence?